.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -14,11 +14,21 @@ jobs:
     strategy:
       matrix:
         java: [ 11, 17 ]
-        bwc_version : [ "1.1.0", "1.2.4", "1.3.8", "2.0.1", "2.1.0", "2.2.1", "2.3.0", "2.4.1", "2.5.0", "2.6.0", "2.7.0", "2.8.0" ]
-        opensearch_version : [ "2.9.0-SNAPSHOT" ]
+        os: [ubuntu-latest]
+        bwc_version : [ "2.0.1", "2.1.0", "2.2.1", "2.3.0", "2.4.1", "2.5.0", "2.6.0", "2.7.0", "2.8.0", "2.9.0-SNAPSHOT" ]
```

Comment:
I dont think we need these.. for 2.x we dont have to check with 3.0 bwc.

Reply:
raised new PR https://github.com/opensearch-project/k-NN/pull/955